packages/e2e-tests/plugins/interactive-blocks/directive-style/block.json

```diff
@@ -0,0 +1,14 @@
+{
+	"apiVersion": 2,
```

Comment:
@DAreRodz is there any reason to use `2` instead of `3` in our e2e blocks?

Reply:
Not really, I think. 🤔 I guess I copy-pasted some of the blocks defined in other plugins inside that folder.

Reply:
Ok, no problem 🙂